Patch by Robert Scheck <robert@fedoraproject.org> for pdftk >= 3.3.3, which replaces the since Bouncy Castle
1.70 deprecated and since 1.75 removed ASN1TaggedObject.getObject() by ASN1TaggedObject.getBaseObject(). See
also:

 - https://gitlab.com/pdftk-java/pdftk/-/issues/155
 - https://github.com/pdftk-java/pdftk-java-container/commit/400123dd3dd25bffc8e19658ff7fc56274da3dd8

--- pdftk-3.3.3/java/com/gitlab/pdftk_java/com/lowagie/text/pdf/PdfPKCS7.java	2022-09-22 19:23:10.000000000 +0200
+++ pdftk-3.3.3/java/com/gitlab/pdftk_java/com/lowagie/text/pdf/PdfPKCS7.java	2024-07-29 23:35:20.739456071 +0200
@@ -225,7 +225,7 @@
         ASN1ObjectIdentifier objId = (ASN1ObjectIdentifier)signedData.getObjectAt(0);
         if (!objId.getId().equals(ID_PKCS7_SIGNED_DATA))
             throw new SecurityException("Not a valid PKCS#7 object - not signed data");
-        ASN1Sequence content = (ASN1Sequence)((DERTaggedObject)signedData.getObjectAt(1)).getObject();
+        ASN1Sequence content = (ASN1Sequence)((DERTaggedObject)signedData.getObjectAt(1)).getBaseObject();
         // the positions that we care are:
         //     0 - version
         //     1 - digestAlgorithms
@@ -258,7 +258,7 @@
         // the possible ID_PKCS7_DATA
         ASN1Sequence rsaData = (ASN1Sequence)content.getObjectAt(2);
         if (rsaData.size() > 1) {
-            DEROctetString rsaDataContent = (DEROctetString)((DERTaggedObject)rsaData.getObjectAt(1)).getObject();
+            DEROctetString rsaDataContent = (DEROctetString)((DERTaggedObject)rsaData.getObjectAt(1)).getBaseObject();
             RSAdata = rsaDataContent.getOctets();
         }
         
@@ -294,7 +294,7 @@
         next = 3;
         if (signerInfo.getObjectAt(next) instanceof ASN1TaggedObject) {
             ASN1TaggedObject tagsig = (ASN1TaggedObject)signerInfo.getObjectAt(next);
-            ASN1Sequence sseq = (ASN1Sequence)tagsig.getObject();
+            ASN1Sequence sseq = (ASN1Sequence)tagsig.getBaseObject();
             ByteArrayOutputStream bOut = new ByteArrayOutputStream();            
             ASN1OutputStream dout = ASN1OutputStream.create(bOut);
             try {
